julia_src/reopt_model.jl

```diff
@@ -2338,7 +2338,7 @@ end
 function add_util_results(m, p, r::Dict)
     net_capital_costs_plus_om = value(m[:TotalTechCapCosts] + m[:TotalStorageCapCosts] + m[:GHPCapCosts]) +
                                 value(m[:TotalPerUnitSizeOMCosts] + m[:TotalPerUnitProdOMCosts] 
-                                    + m[:TotalHourlyCHPOMCosts] + m[:GHPOMCosts]) * m[:r_tax_fraction_owner]
+                                    + m[:TotalHourlyCHPOMCosts] + m[:GHPOMCosts]) * m[:r_tax_fraction_owner] +
```

Comment:
@Bill-Becker would you mind doing a sanity check on this change? Just seems odd that this was missing before

Reply:
wow how did that not error? Yes it needs the plus.